native/general_double.go

```diff
@@ -10,7 +10,17 @@ import (
 	"math"
 )
 
-func checkMatrix(m, n int, a []float64, lda int) {
+func sliceView64(a []float64, lda, i, j, r, c int) []float64 {
+	return a[i*lda+j : (i+r-1)*lda+j+c]
+}
+
+// blocks returns the number of divisons of the dimension length with the given
+// block size.
+func blocks64(dim, bsize int) int {
```

Comment:
Good point. Done.

> On Apr 28, 2016, at 10:40 PM, Dan Kortschak <notifications@github.com> wrote:
> 
> In native/general_double.go <https://github.com/gonum/blas/pull/177#discussion_r61533879>:
> 
> > @@ -10,7 +10,17 @@ import (
> >  	"math"
> >  )
> >  
> > -func checkMatrix(m, n int, a []float64, lda int) {
> > +func sliceView64(a []float64, lda, i, j, r, c int) []float64 {
> > +	return a[i*lda+j : (i+r-1)*lda+j+c]
> > +}
> > +
> > +// blocks returns the number of divisons of the dimension length with the given
> > +// block size.
> > +func blocks64(dim, bsize int) int {
> Can this not go in a common file?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/gonum/blas/pull/177/files/f22d4fe0e2f9e5f0ec7d6b85e42685b77ee193cc..0b9ca4b9d412bf6c79ab202966050c235e581fa5#r61533879>